suggestions/maps.json

```diff
@@ -4,5 +4,12 @@
         "label": "Organic Maps",
         "source": "fg",
         "repo": "https://github.com/organicmaps/organicmaps"
+    },
+    {
+        "id": "net.osmand.plus",
+        "label": "OsmAnd~",
+        "source": "fg",
+        "reason": "Global Mobile Map Viewing & Navigation for Offline and Online OSM Maps. Has telemetry bits that can be disabled in settings",
+        "repo": "https://github.com/osmandapp/Osmand"
```

Comment:
Organic Maps already offers better features than this and does not have any opt-in trackers, which is why it was chosen over OsmAnd~.

Reply:
From my personal experience, Organic Maps is only useful for guided navigation and **basic** places information.

To find useful information about places, having maps available offline, and even to contribute to OpenStreetMap, OsmAnd is far better!

Reply:
Contributing to OSM is not something we focus on simply because most users aren't expected to contribute to OSM. OM is also a lot more faster than OsmAnd~ and looks quite similar to other popular map apps.